### PR TITLE
remove '.' from @INC

### PR DESCRIPTION
**Pull request by ttyS4**
This is the same as in https://github.com/quattor/CCM/pull/181

### CODE REVIEW

**Comment by ttyS4** [comment]
this seems like a jenkins issue rather than issue with the code

**Comment by stdweird** [comment]
retest this please

File: src/main/scripts/ncm-query [hunk 1]
```diff
@@ -133,7 +133,9 @@ toolsuite.
 
 package query;
 
+pop @INC if $INC[-1] eq '.';
```
Comment (stdweird) [inline src/main/scripts/ncm-query]: @ttyS4 wrap this in `BEGIN{}` like the other patches

**Comment by stdweird** [comment]
retest this please

**Comment by stdweird** [comment]
retest this please

**Comment by stdweird** [comment]
retest this please

**Comment by stdweird** [comment]
retest this please

File: src/main/scripts/ncm-query (hunk 1 above)
Comment (ttyS4) [reply src/main/scripts/ncm-query]: Good point I have just updated as that.

**Pull request merged**